eZ/Bundle/EzPublishCoreBundle/Resources/views/default/content/embed.html.twig

```diff
@@ -1,7 +1,6 @@
 {% set content_name=ez_content_name(content) %}
 
-{% if location is defined %}
-    <h3><a href="{{ ez_path(location) }}">{{ content_name }}</a></h3>
-{% else %}
-    <h3><a href="{{ ez_path(content) }}">{{ content_name }}</a></h3>
-{% endif %}
+{% set parameters = {
+    "isEmbed": true
+} %}
+<h3><a href="{{ ez_path(content, parameters) }}">{{ content_name }}</a></h3>
```

Comment:
This risks linking to main location when usage expected it to link to one of the other locations afaik, at least that was probably the intention of making sure to check for specified location variable.